environment/src/main/java/com/sequenceiq/environment/environment/flow/deletion/handler/ResourceEncryptionDeleteHandler.java

```diff
@@ -86,6 +86,7 @@ private void deleteEncryptionResources(EnvironmentDto environmentDto, Environmen
             environmentService.save(environment);
         } catch (Exception e) {
             LOGGER.error("Failed to delete Encryption resources for environment {} with error {}", environment.getName(), e);
+            throw e;
```

Comment:
I guess we need to be more specific like new Cloudbreak...Exception(e, "message")

Reply:
Agreed, `CloudbreakServiceException` could be a good choice.